ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -134,7 +137,7 @@ open class TwoLineTitleView: UIView, TokenizedControlInternal {
     // |  |--titleTrailingImageView (chevron, optional)
     // |--subtitleContainer
     // |  |--subtitleLabel
-    // |  |--subtitleImageView (chevron, optional)
+    // |  |--subtitleImageView (chevron, custom, optional)
```

Comment:
I think `chevron, custom` is making this comment more confusing. Might be better of with just `(optional)`?